src/zos_files/zowe/zos_files_for_zowe_sdk/file_system.py

```diff
@@ -127,14 +151,17 @@ def list(self, file_path_name=None, file_system_name=None):
         list all mounted filesystems, or the specific filesystem mounted at a given path, or the
         filesystem with a given Filesystem name.
 
-        Parameter
+        Parameters
         ---------
-        file_path: str - the UNIX directory that contains the files and directories to be listed.
-        file_system_name: str - the name for the file system to be listed
+        file_path: str
+            the UNIX directory that contains the files and directories to be listed.
+        file_system_name: str
+            the name for the file system to be listed
```

Comment:
Same comment as above about sentence case for consistency
```suggestion
        file_path: str
            USS directory that contains the files and directories to be listed
        file_system_name: str
            Name of the file system to be listed
```